### DISCUSSION TITLE
Ordering by relation model

### DISCUSSION THREAD

**Discussion by xPetricko** [arthurio/fastapi-filter#391] [Q&A] [answered]
Hello, I want to ask if there is possiblity to order by related models.

I have filter like

```python
class MainModelFilter(Filter):
    order_by: Optional[list[str]]
    related_model: Optional[LocaleFilter] = FilterDepends(with_prefix("related_model", RelatedModelFilter))
```

But when i do request using docs with query parameter related_model__order_by=id, the ordering is not working.

I am using SQLAlchemy as ORM.

Is there a possibility to order by  attribute of related model?

**Comment by arthurio** [comment, marked as answer]
Hi @xPetricko I have explored this here: https://github.com/arthurio/fastapi-filter/pull/233. I'm going to merge the 3 PRs I have open over the summer.

**Reply by xPetricko** [reply]
Thanks @arthurio for repsonse and for your work. (y)
